res/css/structures/auth/_Login.scss

```diff
@@ -96,3 +96,10 @@ div.mx_AccessibleButton_kind_link.mx_Login_forgot {
         cursor: not-allowed;
     }
 }
+.mx_Login_spinner {
```

Comment:
I can't find a match for that CSS class, 

https://github.com/matrix-org/matrix-react-sdk/pull/6636/files#diff-4d55f993b98a009668aad5ab3b9223bacf7c0c140cc109d508b050de07a23d94R380 and https://github.com/matrix-org/matrix-react-sdk/pull/6636/files#diff-4d55f993b98a009668aad5ab3b9223bacf7c0c140cc109d508b050de07a23d94R340 both refer to `mx_Login--spinner`

Also we could probably change the name of that CSS class, it doesn't look related to the login page nor a spinner, it's more a utility to centre items vertically and horizontally

Reply:
@gsouquet yeah, commited @SimonBrandner suggestion but didn't do it everywhere.

Reply:
@gsouquet it adds the padding though, which is very case-dependant here.